MsCorePkg/Library/MacAddressEmulationPlatformLibNull/MacAddressEmulationPlatformLibNull.inf

```diff
@@ -0,0 +1,33 @@
+## @file
+#
+# NULL implementation of the platform library to provide hooks for the MAC Address Emulation feature.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 1.27
+  BASE_NAME                      = MacAddressEmulationPlatformLib
```

Comment:
I would suggest to add Null here so if symbols are ever used it is easier to decode usage